src/Components/ModalWindow/ModalWindow.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { createPortal } from 'react-dom';
+import css from './ModalWindow.less';
+
+const modalRoot = document.getElementById('modalWindow');
+
+export class ModalWindow extends React.Component<
+  { children?: JSX.Element, onClose: () => void }, { element: HTMLDivElement }
+> {
+  constructor(props) {
+    super(props);
+    const element = document.createElement('div');
+    element.className = css.modalWindow;
+
+    const closeButton = document.createElement('div');
```

Comment:
It would be better to create an additional component that'll use `ModalWindow` and be responsible for rendering a close button and handling close event

Reply:
fixed